lib/ruby_lsp/ruby_lsp_rails/code_lens.rb

```diff
@@ -32,25 +40,41 @@ module Rails
     #     # ...
     #   end
     # end
-    # ````
+    # ```
     #
     # The code lenses will be displayed above the class and above each test method.
     #
     # Note: When using the Test Explorer view, if your code contains a statement to pause execution (e.g. `debugger`) it
     # will cause the test runner to hang.
+    #
+    # For the following code, assuming the routing contains `resources :users`, a Code Lens will be seen above each
+    # action.
+    #
+    # ```ruby
+    # GET /users(.:format)
```

Comment:
Let's put this above the `def index` line?

Reply:
Done.